tests/unit/pages/Dashboard.spec.js

```diff
@@ -118,6 +118,10 @@ describe("Dashboard.vue", () => {
       expect(wrapper.find('[data-test="retention"]').text()).toBe(
         String(global.dummyPlioMetrics["percent_one_minute_retention"]) + "%"
       );
+
+      expect(wrapper.find('[data-test="surveyQuestionWarning"]').exists()).toBe(
+        global.dummyPlioMetrics["has_survey_question"]
+      );
```

Comment:
would also need a test for the case when there are no survey questions and so, the warning shouldn't be shown